app/views/users/show.html.erb

```diff
@@ -12,7 +12,7 @@
   <strong>Bio:</strong>
   <%= @user.bio %>
 </p>
-<div id="avatar"><%= image_tag(@user.avatar_url) if @user.avatar? %></div>
+<div id="avatar"><%= image_tag(@user.avatar_url(:thumb)) if @user.avatar? %></div>
```

Comment:
@dacur avatar is my field name, and url is just how to render the image. that is actually carrierwave. i installed rmagick in order to generate a thumbnail. 

do you know of another way?